src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -1,11 +1,28 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
 import java.util.ArrayList;
 import java.util.List;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
```

Comment:
Implementing Cloneable and overriding the `clone` method is not necessary unless you have a specific reason to make a deep copy of Movie objects. This could potentially lead to complex issues with entity identity and state, especially when dealing with managed entities in Hibernate.